docs/brisket.createserver.md

```diff
@@ -10,17 +10,29 @@ Use Brisket.createServer to create a Brisket express engine that you can use in
 ## Required Configuration Options
 `createServer` takes parameters so that you can customize your app. Here are the required options:
 
-#### apiHost
-This is the fully qualified path to the api your Brisket app will hit. This value will be prepended to all model and collection urls during server-side requests.
+#### apis
+Use apis configuration to specify the apis that your application will hit:
+
+```js
+apis: {
+    'api': { host: 'http://api.example.com', proxy: 'http://proxy.example.com' }
+    'other-api': { host: 'http://other-api.example.com' }
+}
+```
+
+For each api, you can specify the host and proxy. Your app's `Backbone.Model`s and `Backbone.Collection`s can [fetch data from the apis you specify](modeling.md#fetch-data).
```

Comment:
@claquesous would this line be enough: `Brisket automatically adds a middleware (relative to your app's root) for each api to handle client side ajax calls.`?